CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [5.0.0] - 2022-11-25
+### Removed
+
+* Removed guzzle5-adapter and guzzle6-adapter documentation and support; this version onwards now supports and recommends only guzzle7-adapter.
```

Comment:
Is this saying in terms of versions of the guzzle adaptor, we only support guzzle7? Or is it saying that in terms of any PSR-7 compatible HTTP client, we only support Guzzle 7? Or maybe these are even the same thing depending on what other HTTP clients are out there?

Reply:
I believe that it should support any PSR-7 compatible library, but we pick and recommend guzzle7 as an example lib to use. Maybe this should be updated to say `this version onwards now supports any PSR-7 compatible library and provides documentation for how to use it with guzzle7`? 🤔 @danielcarney96?

Reply:
It's entirely based on the PSR-7 compatibility so yeah it might be good to clarify that.